core/src/tile/tileManager.cpp

```diff
@@ -6,32 +6,30 @@
 #include "tile/mapTile.h"
 #include "view/view.h"
 
+#include "glm/gtx/norm.hpp"
+
+#include <chrono>
 #include <algorithm>
 
-TileManager::TileManager() {
+#define DBG(...)
```

Comment:
Maybe define this in the platform header.

We also have a trello card to expose debugging toggles.